Sources/PrimerSDK/Classes/Data Models/PaymentMethodConfigurationType.swift

```diff
@@ -241,6 +248,7 @@ public enum PaymentMethodConfigType: Codable, Equatable {
         case googlePay
         case hoolah
         case klarna
+        case mbWay
```

Comment:
Is `mbWay` linked with Blik somehow? It seems another type of PM so I wonder if we need to add anything in the above `PaymentMethodConfigType` `rawValue` switch?